manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -148,7 +148,7 @@ spec:
           - name: "TAINT_CONFIGMAP-NAME"
             value: "istio-cni-taint-configmap"
           - name: "TAINT_CONFIGMAP-NAMESPACE"
-            value: "kube-system"
+            value: {{ .Release.Namespace | quote }}
```

Comment:
Deploying into `kube-system` was necessary because of the `priorityClassName: system-cluster-critical`. Maybe this constraint has been relaxed in more recent versions of k8s (???), but this will certainly break on older k8s versions.

Reply:
@rlenglet We use a custom namespace to deploy istio-cni. Our daemonsets are marked as `system-node-critical` which is higher than `system-cluster-critical`.

Deploying into `kube-system` is not an option for us.  I believe this might be a limitation in kubernetes versions 1.15 and lower. 

Is there any plan to retire k8s 1.15 support at all?